README.md

```
@@ -0,0 +1,44 @@
+```
+
+Quick start (model API, with Docker)
+
+```bash
+docker build -t sentiment-model -f model-lab/Dockerfile model-lab
+docker run -p 8080:8080 sentiment-model
+# API will be available at http://localhost:8080
+```
+
+Usage
+- Demo UI: http://localhost:3000/demo
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)